build-logic/src/main/java/qsl/internal/extension/QslExtension.java

```diff
@@ -4,9 +4,7 @@
 import qsl.internal.Versions;
 
 public class QslExtension {
-	private static final String RESOLVE_VINEFLOWER_TASK = "resolveVineflower";
-	private static final String GEN_SOURCES_WITH_VINEFLOWER_TASK = "genSourcesWithVineflower";
-	private static final String[] TASKS_TO_DISABLE = {RESOLVE_VINEFLOWER_TASK, "genSourcesWithFernFlower"};
+	private static final String[] TASKS_TO_DISABLE = {"genSourcesWithFernFlower"};
```

Comment:
Why remove the exclusion? I believe we'll still need it in order to prevent the silly case of Decompiling Everything

Reply:
Loom 1.4 adds vineflower support so i can remove the special casing that we do for it

Reply:
Yes, I can see why the "make `genSourcesWithVineflower` depend on `resolveVineflower`" part was removed, but what I don't understand is why it was removed from `TASKS_TO_DISABLE`;
It is there in order to prevent a `./gradlew genSources` task from decompiling each module's Minecraft, so it still has to be there

Reply:
This task existed on loom 1.3 too

Reply:
Ah ok i see we disabled the resolve task which then in turn disables the gen sources task. ill fix this